.github/workflows/Tutor_CI.yml

```diff
@@ -0,0 +1,32 @@
+name: Tutor_CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
```

Comment:
I'd try to share the code below with your Vagrant environment - perhaps run `bootstrap.sh` instead of re-doing what it does already?